intake/cli/server/tests/test_server.py

```diff
@@ -257,6 +257,9 @@ def multi_server(tmpdir):
     shutil.rmtree(tmpdir)
 
 
+# If this tests fails on Mac running Monterey or newer,
+# Look at this solution to Apple using port 5000:
+# https://developer.apple.com/forums/thread/682332
 def test_flatten_flag(multi_server):
```

Comment:
I suppose in the long run we ought to have a fixture picking random ports and retrying if the port is not available. Now now, though.